fish_morphology_code/processing/structure_organization/tools/features.py

```diff
@@ -39,7 +39,7 @@ def process_fov(FOVId, df_fov):
         total_area = mask.sum()
 
         # Classification results from CNN classifier
-        classification = data[ch_cla].astype(np.uint8) - 1
+        classification = data[ch_cla].astype(np.uint8)
```

Comment:
what does this off by one error fix?

Reply:
To make sure the code run when we have background class that has label 0.